contracts/connectors/loantoken/modules/LoanTokenLogicStandard.sol

```diff
@@ -417,6 +417,18 @@ contract LoanTokenLogicStandard is LoanTokenLogicStorage {
 			);
 	}
 
+	/**
+	 * @notice Withdraws RBTC from the contract by Multisig.
+	 * @param _receiverAddress The address where the rBTC has to be transferred.
+	 * @param _amount The amount of rBTC to be transferred.
+	 */
+	function withdrawRBTCTo(address payable _receiverAddress, uint256 _amount) external onlyOwner {
+		require(_receiverAddress != address(0), "receiver address invalid");
+		require(_amount > 0, "non-zero withdraw amount expected");
+		require(_amount <= address(this).balance, "withdraw amount cannot exceed balance");
+		_receiverAddress.transfer(_amount);
```

Comment:
consider emitting an event to simplify monitoring.

Reply:
Done